test/dotcom/utils/service_date_time_test.exs

```diff
@@ -226,6 +226,38 @@ defmodule Dotcom.Utils.ServiceDateTimeTest do
     end
   end
 
+  describe "service_range_range/2" do
+    test "returns all service ranges between two datetimes" do
+      # Setup
+      today = service_range_day() |> random_time_range_date_time()
+      next_week = service_range_next_week() |> random_time_range_date_time()
+
+      # Exercise / Verify
+      assert service_range_range(today, next_week) == [:today, :this_week, :next_week]
+    end
+
+    test "handles open end" do
```

Comment:
This could be more descriptive in line with the actual assertion. Also, we should call it 'stop' rather than 'end.'

Reply:
Good call, I've [changed this](https://github.com/mbta/dotcom/pull/2403/commits/5f2c060c769962c97362ec478dfdb0e489e7e265) to describe more directly what's happening.